module/zfs/mmp.c

```diff
@@ -429,6 +430,9 @@ mmp_thread(void *arg)
 		 */
 		if (!suspended && mmp_fail_intervals && multihost &&
 		    (start - mmp->mmp_last_write) > max_fail_ns) {
+			cmn_err(CE_WARN, "MMP writes to pool '%s' have not "
```

Comment:
I think you should mention that the pool will be suspended in this message. This will help connect the dots if the system does end up panicking due to the failure mode.

How about something like `MMP writes to pool 'name' have not succeeded in over Xs. The pool will be suspended.`?

Reply:
Well, regardless of whether the system panics or not, zio_suspend() will emit a message that the pool is being suspended. But I can add what you suggest to the new message if that's what you want.

I'm not sure how to address the style check failure. Running 'make checkstyle' locally passes. The errors listed http://build.zfsonlinux.org/builders/Ubuntu%2017.10%20x86_64%20%28STYLE%29/builds/1998/steps/shell_2/logs/stdio don't seem to have anything to do with this change. Any hints?

Reply:
@jhammond-intel I think this makes sense as a quick fix.  Longer term I think it would be nice to pass a reason to `zio_suspend()` so it can print a more descriptive error message.

Go ahead and ignore the style check failures for this one.  They're not your fault.  I updated the VM running the checks last week and pulled in a new version of `shellcheck` which appears to be upset about new issue which haven't yet been addressed/suppressed.  See #7040 (which still needs work).